src/com/collection/MyLinkedList.java

```diff
@@ -0,0 +1,104 @@
+package collection;
+
+public class MyLinkedList<T> implements List<T> {
+    private Node<T> first;
+    private Node<T> last;
+    private int size;
+
+    public MyLinkedList() {
+        first = new Node<>(null, null, null);
+        last = first;
+        size = 0;
+    }
+
+    //addLast
+    @Override
+    public void add(T item) {
+        Node<T> newNode = new Node<>(last, item, null);
+        if (size == 0) {
+            first = newNode;
+        }
+        last.next = newNode;
```

Comment:
It looks like a singly linked list.
Please, read https://habr.com/ru/post/127864/
You don't use Node.prev